spec/bourbon/addons/buttons_spec.rb

```diff
@@ -4,49 +4,50 @@
   before(:all) do
     ParserSupport.parse_file("addons/buttons")
 
-    @buttons_list = [
-      "button",
-      "input[type=\"button\"]",
-      "input[type=\"reset\"]",
-      "input[type=\"submit\"]"
-    ]
+    @buttons_list = %w(
+      button
+      input[type="button"]
+      input[type="reset"]
+      input[type="submit"]
+    )
   end
 
   context "expands plain buttons" do
     it "finds selectors" do
-      @buttons_list.each do |input|
-        expect(input).to have_rule("color: #ff0000")
-      end
+      list = @buttons_list.join(", ")
+      ruleset = "value: #{list};"
+
+      expect("#all-button-inputs").to have_ruleset(ruleset)
```

Comment:
While this is still valid (via a fallback), we tweaked the name of this variable to simply `$all-buttons` in https://github.com/thoughtbot/bourbon/commit/a6374b6be375737cdfcb081d3a668b4865692038.